apps/game_backend/lib/game_backend/curse_of_mirra/quests.ex

```diff
@@ -109,8 +109,13 @@ defmodule GameBackend.CurseOfMirra.Quests do
   """
   def get_user_quest(user_quest_id) do
     q = from(uq in UserQuest, preload: [:quest], where: uq.id == ^user_quest_id)
+    user_quest = Repo.one(q)
 
-    Repo.one(q)
+    if user_quest do
+      {:ok, user_quest}
+    else
+      {:error, :not_found}
+    end
```

Comment:
Use is_nil and maybe a case statement. Also the user_quest variable isn't needed.

Reply:
I used a case statement with a nil case